common/djangoapps/student/views.py

```diff
@@ -10,6 +10,7 @@
 from pytz import UTC
 from requests import HTTPError
 from ipware.ip import get_ip
+from urlparse import urljoin
```

Comment:
Minor: `urlparse` is part of the standard library; this should be placed with other standard library imports above.